docs/source/_json/collection.resp

```diff
@@ -16,7 +16,7 @@ Content-Type: application/json
   "@id": "http://localhost:55001/plone/collection", 
   "@type": "Collection", 
   "UID": "SomeUUID000000000000000000000002", 
-  "allow_discussion": null, 
+  "allow_discussion": false, 
```

Comment:
Did something change in the types as well?

Reply:
@sneridagh @jaroel please note that "null" is a different setting than "false". "Null" means "check parent object setting" and "false" means "do not allow discussion".

Reply:
Ok, got it. https://docs.google.com/document/d/1vKmHXzw2vVWYEVEvFtIUr7gcpf_3Qh7dhYx0cx5rUGM/edit#heading=h.xr2un7csqeq2 sets global discussion to False, therefore all objects return "False" by default.

Reply:
@tisto where is this set?

Reply:
@tisto Ah yeah, sorry, that makes sense. This looks clean imho.